querysql/querysql.go

```diff
@@ -40,11 +42,16 @@ type ResultSets struct {
 	// By default it is set by New to the value provided by Logger(ctx), but feel free to set or change it.
 	Logger RowsLogger
 
-	// By default, the use of an underscore column, "select _=1, ...", will trigger logging
+	// TODO(dsf): Perhaps remove the LogKeyLowercase.  I'm not 100% this feature is justified
```

Comment:
I'm leaving this comment to myself here and will get back to it later.  I'm leaning towards removing this bit of code, but want to see what the blast radius is first.